commit 927f384a9bc058eb6f954e93cc515cc1293fd2a3
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Fri Oct 17 08:11:10 2008 +0300

    Kick out the dumb defaultdocdir logic from installplatform
    - defaultdocdir is simply %{_datadir}/doc

diff --git a/installplatform b/installplatform
index 8861df0..266d877 100755
--- a/installplatform
+++ b/installplatform
@@ -8,12 +8,6 @@ RPMRC="${1:-rpmrc}"
 MACROS="${2:-macros}"
 PLATFORM="${3:-platform}"
 
-if grep /share/ $PLATFORM > /dev/null 2>&1 ; then
-    DEFAULTDOCDIR='%{_usr}/share/doc'
-else
-    DEFAULTDOCDIR='%{_usr}/doc'
-fi
-
 TEMPRC="/tmp/rpmrc.$$"
 cat << E_O_F > $TEMPRC
 include:	$RPMRC
@@ -155,7 +149,6 @@ for SUBST in $SUBSTS ; do
 	-e "s,@RPMRC_GNU@,$RPMRC_GNU," \
 	-e "s,@LIB@,$LIB," \
 	-e "s,@ARCH_INSTALL_POST@,$ARCH_INSTALL_POST," \
-	-e "s,@DEFAULTDOCDIR@,$DEFAULTDOCDIR," \
 	-e '/\${\w*:-/!s,\${,%{_,' \
 	-e "s,@ISANAME@,$ISANAME," \
 	-e "s,@ISABITS@,$ISABITS," \
diff --git a/platform.in b/platform.in
index 4b496a0..9768a8a 100644
--- a/platform.in
+++ b/platform.in
@@ -37,7 +37,7 @@
 # Deprecated misspelling, present for backwards compatibility.
 %_initrddir		%{_initddir}
 
-%_defaultdocdir		@DEFAULTDOCDIR@
+%_defaultdocdir		%{_datadir}/doc
 
 %_smp_mflags %([ -z "$RPM_BUILD_NCPUS" ] \\\
 	&& RPM_BUILD_NCPUS="`/usr/bin/getconf _NPROCESSORS_ONLN`"; \\\
